src/ocm/components/clusterConfiguration/ReviewCluster.tsx

```diff
@@ -45,6 +55,7 @@ const PlatformIntegrationNote = () => {
 const ReviewCluster = ({ cluster }: { cluster: Cluster }) => {
   const clusterWizardContext = useClusterWizardContext();
   const { t } = useTranslation();
+  const { supportedPlatformIntegration } = useClusterSupportedPlatforms(cluster.id);
```

Comment:
`useClusterSupportedPlatforms` is not the appropriate value here.

It means "the platform which is possible to integrate", whereas in the context of reviewing a cluster, we only want the "platform which has been selected to be integrated".

Meaning, looking at `cluster.platform.type` is enough, and this avoids having to perform extra calls to the API endpoint for feature-support etc.

Ideally, we don't want to use `cluster.platform.type` directly, but through a reusable selector.